File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/Login/RegisterRequest.java
package com.example.APIRollTheDice.Model.DTO.Login;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RegisterRequest {
    @JsonProperty("Username")
    private String username;
    @JsonProperty("Password")
    private String password;
    @JsonProperty("Email")
    private String email;

    public RegisterRequest() {
    }

    public RegisterRequest(String username, String password, String email) {
        this.username = username;
        this.password = password;
        this.email = email;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/Login/RegisterRequest.cs
using System;

namespace Assets._Project.API.Model.DTO.Login
{
    [Serializable]
    public class RegisterRequest
    {
        public string Username { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }

        public RegisterRequest(){}
        public RegisterRequest(string username, string password, string email){
            Username = username;
            Password = password;
            Email = email;
        }
    }
}